EternalModLoader/BlangString.cs

```diff
@@ -0,0 +1,29 @@
+namespace EternalModLoader
+{
+    /// <summary>
+    /// BlangString class
+    /// </summary>
+    public class BlangString
```

Comment:
This shouldn't be here. You should reference the BlangParser library that you can find here: https://github.com/dcealopez/BlangParser

This way, if I implement changes to the BlangParser library, they will be automatically traslated to the mod loader.

Reply:
There is a problem with this though, the methods are modified.